index.html

```diff
@@ -58,6 +57,158 @@ <h6 class="social-media-text">LET’S CONNECT</h6>
 				</ul>
 			</section>
 		</section>
+		<section class="works-section">
+			<ul class="works">
+				<li class="tonic">
+					<img src="./Images/pages/SnapshootPortfolio.png" alt="1" class="image-responsive">
+					<h2>Tonic</h2>
+					<ul class="work-info">
+						<li class="canopy">CANOPY</li>
+						<li><span class="span"></span></li>
+						<li class="canopy1">Back End Dev</li>
+						<li><span class="span"></span></li>
+						<li class="canopy1">2015</li>
+					</ul>
+					<p class="despription">A daily selection of privately personalized reads; no accounts or
+						sign-ups required.</p>
+					<ul class="tech-info">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<button class="btn" type="button">See Project</button>
```

Comment:
- [ ] It would be great if the button follows the Figma design when is active and pressed (hover not at this moment, because on the mobile size we don't use a mouse). 🐱 
![image](https://user-images.githubusercontent.com/20563252/125117526-fd65b100-e0b3-11eb-876f-72693785f9bd.png)